### PR TITLE
Server rules

### LINKED ISSUE #21272: server.rules_file shouldn't be hardcoded to have a root of ServerInfo/
## Description
Hardcoding which part of the VFS to read is simply cringe.

## Proposed solution
https://github.com/space-wizards/space-station-14/blob/f4c3d66b1c2ecad9a1799cd71a7adca4fab4551d/Content.Server/Info/InfoSystem.cs#L27 change this scrunkly to not be scrunkly

### PR DESCRIPTION
**Pull request by dakamakat**
## About the PR

resolves #21272 

## Technical details

Change CCVars to not be so scrunky

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by deltanedas** [comment]
this might be breaking for downstream

**Comment by dakamakat** [comment]
@deltanedas  there is no other dependencies on that path variable and it's not changing actual directory not quite sure how it's supposed to break smth

**Comment by deltanedas** [comment]
as in if a server has path set to sussy.txt it used to look for /ServerInfo/sussy.txt but it will now look for /sussy.txt which might break rules

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -18,10 +18,10 @@ public sealed class CCVars : CVars
             CVarDef.Create("server.id", "unknown_server_id", CVar.REPLICATED | CVar.SERVER);
 
         /// <summary>
-        ///     Name of the rules txt file in the "Resources/Server Info" dir. Include the extension.
+        ///     Path to the rules txt file in the "Resources/Server Info" dir. Include the extension.
         /// </summary>
-        public static readonly CVarDef<string> RulesFile =
-            CVarDef.Create("server.rules_file", "Rules.txt", CVar.REPLICATED | CVar.SERVER);
+        public static readonly CVarDef<string> RulesPath =
+            CVarDef.Create("server.rules_file", "/ServerInfo/Rules.txt", CVar.REPLICATED | CVar.SERVER);
```
Comment (dakamakat) [inline Content.Shared/CCVar/CCVars.cs:24]: @deltanedas I just move /ServerInfo/ to CVar  , and if i understood ur example well it still be the same path with /ServerInfo/ as it supposed to be

Comment (dakamakat) [reply Content.Shared/CCVar/CCVars.cs:24]: Oh , i understood , you're talking about values that goes from toml presets to this variables

**Comment by metalgearsloth** [comment]
Splitting it into 2 cvars is also not ideal, better to just have the 1 cvar.

**Comment by dakamakat** [comment]
@metalgearsloth actually the only way i see to address original issue #21272 is to split it to two variables , if we have one cvar then adressing original issue we need to specify actual path to rules file in server configuration

**Comment by VasilisThePikachu** [comment]
The problem here was that the path of the rules file was hardcoded. Even if it is 1 cvar. You can still manually specify the correct path

You can make the default for this cvar "/ServerInfo/Rules.txt" for example (wherever the rule file is)

**Comment by metalgearsloth** [comment]
@dakamakat are you coming back to this.

**Comment by dakamakat** [comment]
No , i 'll close this

**Pull request closed without merging**